src-java/flowhs-topology/flowhs-storm-topology/src/main/java/org/openkilda/wfm/topology/flowhs/model/path/FlowPathReference.java

```diff
@@ -0,0 +1,30 @@
+/* Copyright 2022 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.wfm.topology.flowhs.model.path;
+
+import org.openkilda.model.PathId;
+
+import lombok.NonNull;
+import lombok.Value;
+
+@Value
+public class FlowPathReference {
```

Comment:
Please, reconsider the naming.

Reply:
I would like to get some hint here. The goal of this object is to define and path related to specific flow.